e2e/app/src/main/java/com/example/app/env/EnvViewModel.kt

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2023 - 2024 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package com.example.app.env
+
+import android.content.Context
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.app.storage.loadCookiesStorage
+import com.example.app.storage.loadSSOTokenStorage
+import com.example.app.storage.loadTokenStorage
+import kotlinx.coroutines.launch
+import org.forgerock.android.auth.ContextProvider
+import org.forgerock.android.auth.FRAuth
+import org.forgerock.android.auth.FROptions
+import org.forgerock.android.auth.FROptionsBuilder
+
+const val USER_PROFILE_JOURNEY = "UserProfile"
+
+class EnvViewModel : ViewModel() {
+
+    val servers = mutableListOf<FROptions>()
+
+    val localhost = FROptionsBuilder.build {
+        server {
+            url = "https://andy.petrov.ca/openam"
```

Comment:
Not sure if we should keep these environments in the sample app... 
I think our docs has some references to this file... Let's confirm with @forgerock-chris though...

Reply:
I agree, we should remove those. Ideally we should load these configs from a file, and ignore this file on commits.

Reply:
Revert to sample env.